apps/prairielearn/src/pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js

```diff
@@ -17,23 +17,27 @@ const sql = sqldb.loadSqlEquiv(__filename);
 
 const ensureUpToDate = (locals, callback) => {
   debug('ensureUpToDate()');
-  assessment.update(locals.assessment_instance.id, locals.authn_user.user_id, (err, updated) => {
-    if (ERR(err, callback)) return;
+  util.callbackify(assessment.update)(
+    locals.assessment_instance.id,
+    locals.authn_user.user_id,
+    (err, updated) => {
```

Comment:
This is the only reference to this function. I didn't want to convert this to async in this PR as the code is a bit complex to do just a partial change, but I kept the reference using `util.callbackify` so that a future revamp of this code has an easier time converting this function.